### PR TITLE
feature/drop-35/onboarding glucose notification list

### PR DESCRIPTION
**Pull request by Aivaxela**
## Describe your changes

Added component for creating reminders at the "Glucose Notifications List" step. Component currently logs an array of times selected. Integrated "add-more-section" component.

## Issue ticket number and link

DROP-35
https://tripleten-apiary.atlassian.net/browse/DROP-35?atlOrigin=eyJpIjoiNDcyZGM5NGQ3MTcyNGViNDg0MTM2ZmQ2NjYzOTVhOTUiLCJwIjoiaiJ9

## Checklist before requesting a review

- [x] I have performed a self-review of my code
- [ ] If there are changes to components, I have added / updated automated tests
- [x] I have made any necessary updates to Storybook to accompany these changes
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings

### CODE REVIEW

File: src/routes/onboarding/set-medication-reminders/medication-reminder-form.tsx [hunk 1]
```diff
@@ -0,0 +1,17 @@
+import React from "react";
+import { Form } from "../../../components"
+import { MedicationSection } from "./medication-section";
+
+export const MedicationReminderForm: React.FC = () => {
+  return (
+    <div className="flex flex-col bg-lightYellow max-w-screen-md pb-8 px-[10px] relative m-auto">
+      <h2 className="font-text text-section-header font-medium leader-[38.4px] text-center max-w-[358px] mx-auto">Set your medication reminders</h2>
```
Comment (Brian-J-Wang) [inline src/routes/onboarding/set-medication-reminders/medication-reminder-form.tsx:8]: Good use of header and paragraph heirarchy

File: src/routes/onboarding/set-medication-reminders/medication-reminder-form.tsx [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+      <p className="font-text text-center max-w-[358px] mx-auto mt-4 mb-8">It’s important to be consistent in your medications, and notifications may help!</p>
+      <Form className="flex flex-col gap-5 ">
+        <Form.AddMoreSection buttonText="+ Add another medication">
```
Comment (Brian-J-Wang) [inline src/routes/onboarding/set-medication-reminders/medication-reminder-form.tsx:11]: The spacing here between the button and the bottom medication is also too much.

The Figma docs
image

Your Implementation
image

**Review by Brian-J-Wang — CHANGES_REQUESTED** [review]
Everything works and looks good to me. Just some spacing issues with the buttons and the medication reminders.

**Review by Brian-J-Wang — APPROVED** [review]
Looks good to me.

File: src/routes/onboarding/glucose-notification-list.tsx [hunk 3]
```diff
@@ -0,0 +1,24 @@
...
+
+interface Props {
+  index?: number;
+  onDelete?: (index: number) => void;
+  elementId?: number;
+}
+
+export const GlucoseNotificationList: React.FC<Props> = ({ index }) => {
+  return (
+    <div className="flex flex-col max-w-screen-md bg-lightPink relative overflow-hidden m-auto pb-80 h-screen">
```
Comment (CalebHasch) [inline src/routes/onboarding/glucose-notification-list.tsx:11]: After adding a few reminders they go off screen and you can't scroll down. If you add min-h-fit that should make sure the div is always bigger than the form and fix that problem. Also I don't think you need all that top and bottom padding.

Comment (Aivaxela) [reply src/routes/onboarding/glucose-notification-list.tsx:11]: Fixed the scrolling issue and removed the padding - the outermost div is really only for conceptualizing the component at the onboarding steps and may be removed anyways depending on how the onboarding wizard is implemented.

**Review by CalebHasch — APPROVED** [review]
Looks awesome! You're approved for final review

File: src/main.tsx [hunk 4]
```diff
@@ -16,13 +22,23 @@ ReactDOM.createRoot(document.getElementById("root")!).render(
...
+            path="/registration-confirm"
+            element={<RegistrationConfirmation />}
+          />
+          <Route
+            path="/onboarding/glucose-notifications"
+            element={<GlucoseNotificationPrompt />}
+          />
+          <Route
+            path="/onboarding/glucose-notifications-list"
+            element={<GlucoseNotificationList />}
```
Comment (RPinkha) [inline src/main.tsx:35]: Small issue here, the ticket mentions this to be a component - evident by the tsx file missing the logo and the back button, so there is no need to route it here in main.

File: src/routes/onboarding/glucose-notification-list.tsx [hunk 5]
```diff
@@ -0,0 +1,24 @@
...
+interface Props {
+  index?: number;
+  onDelete?: (index: number) => void;
+  elementId?: number;
+}
+
+export const GlucoseNotificationList: React.FC<Props> = ({ index }) => {
+  return (
+    <div className="flex flex-col max-w-screen-md bg-lightPink overflow-y-auto relative m-auto h-screen">
+      <div className="flex flex-col self-center gap-6 items-center px-4 max-w-[500px]">
```
Comment (RPinkha) [inline src/routes/onboarding/glucose-notification-list.tsx:12]: Figma shows a gap of 32px. "gap-8" would work better here.

File: src/routes/onboarding/glucose-notification-list.tsx [hunk 6]
```diff
@@ -0,0 +1,24 @@
...
+        <h2 className="font-text text-section-header font-medium leading-[38.4px] text-center max-w-[358px] mx-auto">
```
Comment (RPinkha) [inline src/routes/onboarding/glucose-notification-list.tsx:13]: I might be off here, but the color needs to be #121212. 
Add text-black to achieve that.

**Review by RPinkha — COMMENTED** [review]
A couple of minor issues to fix. 
Looks amazing otherwise.

**Review by jahorwitz — APPROVED** [review]

**Pull request merged**